dbms/src/Flash/Disaggregated/WNEstablishDisaggTaskHandler.cpp

```diff
@@ -54,15 +54,6 @@ void WNEstablishDisaggTaskHandler::prepare(const disaggregated::EstablishDisaggT
     auto start_ts = meta.start_ts();
     context->setSetting("read_tso", start_ts);
 
-    if (request->timeout_s() < 0)
-    {
-        throw TiFlashException(Errors::Coprocessor::BadRequest, "invalid timeout={}", request->timeout_s());
-    }
-    else if (request->timeout_s() > 0)
-    {
-        context->setSetting("disagg_task_snapshot_timeout", request->timeout_s());
-    } // use default timeout if it is 0
-
```

Comment:
Delete these rows so that we can define disagg_task_snapshot_timeout by wn config file instead of rn